src/test/java/com/ksm/wstbackoffice/controller/AddressControllerTest.java

```diff
@@ -0,0 +1,92 @@
+package com.ksm.wstbackoffice.controller;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.http.ContentType.JSON;
+import static org.hamcrest.Matchers.is;
+
+@SpringBootTest
+public class AddressControllerTest {
+    @Value("${server.port}")
+    private int port;
+
+    @Value("${server.address}")
+    private String address;
+
+    @Value("${local.protocol}")
+    private String protocol;
+
+    @Test
+    public void findByIdTest() {
+        given()
+                .accept(JSON)
+                .pathParam("id", 1)
+                .when()
+                .get(String.valueOf(new StringBuilder()
+                        .append(protocol)
+                        .append(address)
+                        .append(":")
+                        .append(port)
+                        .append("/addresses/{id}")))
+                .then()
+                .statusCode(200)
+                .body(
+                        "id", is(1)
+                );
+    }
+
+    @Test
+    public void findAllTest() {
+        given()
+                .when()
+                .get(String.valueOf(new StringBuilder()
+                        .append(protocol)
+                        .append(address)
+                        .append(":")
+                        .append(port)
+                        .append("/addresses")))
+                .then()
+                .statusCode(200)
+                .log()
+                .all();
+    }
+
+    @Test
+    public void addTest() {
```

Comment:
To start you should use as a test method name the real method name + Test.
E.g. for method update -> updateTest.
Do you have method add?

Reply:
Yes, I have - In  AddressController
@PostMapping
public ResponseEntity<Address> add(@RequestBody Address entityAddress) {

Reply:
Please rename the method to create

Reply:
done